projects/demo/src/app/components/csv-entry/csv-entry.component.ts

```diff
@@ -1,5 +1,6 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
```

Comment:
..... just in demo though

Reply:
Yeah when we got rid of TSLint I relaxed some style-based rules in the demo and VR apps